Dockerfile

```diff
@@ -34,10 +35,21 @@ ENV OUTPUT_CLOUDWATCH_ENABLED   false
 ENV OUTPUT_KAFKA_ENABLED        false
 
 COPY telegraf.conf.tpl /etc/telegraf/telegraf.conf.tpl
-COPY run.sh /run.sh
```

Comment:
@freignat91 I see. what's the purpose of the first loop by the way? is it only to restart telegraf in case it crashes? isn't that the role of the orchestrator?